splice_access_api/src/main/java/com/splicemachine/access/configuration/OlapConfigurations.java

```diff
@@ -117,6 +117,10 @@
     public static final String SPARK_COMPACTION_BLOCKING = "spark.compaction.blocking";
     public static final boolean DEFAULT_SPARK_COMPACTION_BLOCKING = true;
 
+    // Kafka Bootstrap Servers
+    public static final String KAFKA_BOOTSTRAP_SERVERS = "splice.kafka.bootstrapServers";
```

Comment:
The same parameter is defined twice in HbaseConfigurations & OlapConfigurations, we probably only need one

Reply:
Removed param from OlapConfigurations in new commit [ce45a6e](https://github.com/splicemachine/spliceengine/pull/3391/commits/ce45a6e2c9126ce86c5a6b350bfbbaffe4ca2a5c)